debian/control

```diff
@@ -0,0 +1,16 @@
+Source: mercury
+Maintainer: Team Kano <dev@kano.me>
+Section: tools
+Priority: optional
+Standards-Version: 1.1.0
+Build-Depends: debhelper(>= 9.0.0), build-essential, cmake, python-conan, lcov
+
+Package: mercury
+Architecture: any
+Depends: libc6, libstdc++6, libgcc1
+Description: Kano OS multiplatform API abstraction layer
+
+Package: mercury-dev
+Architecture: any
+Depends: mercury
```

Comment:
Might want this dependency to require the same version of `mercury` as the one provided by this same package as the version of the headers is going to be heavily tied into the underlying version.

```suggestion
Depends: mercury  (= ${binary:Version})
```

Reply:
That's a good point, thanks! fixed and building fine.